src/client/gui/lib/providers.dart

```diff
@@ -138,6 +145,12 @@ final deletedVmsProvider = Provider((ref) {
       .toBuiltSet();
 });
 
+final zonesProvider = Provider<AsyncValue<BuiltList<Zone>>>((ref) {
+  return ref
+      .watch(pollingProvider)
+      .whenData((data) => data.zones.build() as BuiltList<Zone>);
+});
```

Comment:
Again, as I mentioned in my previous comment, I think we should drop the `AsyncValue` stuff. Please refer to my original comment on this piece of code https://github.com/canonical/multipass/pull/3979#discussion_r2036062468. Of course, feel free to discuss this and provide counter-arguments :)
```suggestion
final zonesProvider = Provider<BuiltList<Zone>>((ref) {
  return ref.watch(pollingProvider).valueOrNull?.zones.build() ?? BuiltList();
});
```